packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -114,144 +126,168 @@ export const ContentCard = ({
 
   return (
     <Box.div
-      as={isCardInteractive ? "a" : as}
-      backgroundColor={backgroundColor[background]}
       border={0}
-      borderRadius={"borderRadius40"}
+      borderRadius="borderRadius40"
       boxShadow={
-        isCardInteractive && {
-          _: "none",
-          hover: "shadowStrong",
-        }
-      }
-      display="flex"
-      flexDirection={
-        isImageOnTop ? "column-reverse" : { _: "column-reverse", md: "unset" }
+        isCardInteractive
+          ? {
+              _: "none",
+              hover: "shadowStrong",
+            }
+          : "none"
       }
-      fontFamily="fontFamilyNotoSans"
-      justifyContent="space-between"
       maxH={maxHeight[imagePosition]}
       maxW={maxWidth || maxWidthDefault[imagePosition]}
-      padding="d0"
-      textDecoration="none"
-      {...(isCardInteractive ? interactiveElementProps : {})}
+      overflow="hidden"
+      position="relative"
     >
       <Box.div
-        borderRadius="borderRadius40 borderRadius0 borderRadius0 borderRadius40"
-        lineHeight="lineHeight30"
-        maxWidth={isImageOnTop ? "100%" : { _: "100%", md: "50%" }}
-        padding="d6"
+        as={isCardInteractive ? "a" : as}
+        backgroundColor={backgroundColor[background]}
+        display="flex"
+        flexDirection={
+          isImageOnTop ? "column-reverse" : { _: "column-reverse", md: "unset" }
+        }
+        fontFamily="fontFamilyNotoSans"
+        h={h}
+        justifyContent="start"
+        maxH={maxHeight[imagePosition]}
+        maxW={maxWidth || maxWidthDefault[imagePosition]}
+        opacity={lowOpacity ? "0.6" : "1"}
```

Comment:
```suggestion
        opacity={lowOpacity ? 0.6 : 1}
```

Reply:
Changed it here https://github.com/Localitos/pluto/pull/1752/files#diff-f7fa8c6df160defa8d1aad22a1a3e5af7da525824cca21b138dee5363ceedbd9R156